deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -114,6 +114,12 @@ objects:
             exec:
               command:
               - "/opt/apache-cassandra/bin/cassandra-docker-ready.sh"
+          lifecycle:
+            preStop:
+              exec:
+                command:
+                - "/opt/apache-cassandra/bin/cassandra-prestop.sh"
+          terminationGracePeriodSeconds: 1800
```

Comment:
I wonder if this value is available via the downward API.  It would be nice to output the value in the stop message to the user so when they ask why its taking so long.